spec/models/guest_spec.rb

```diff
@@ -0,0 +1,17 @@
+require "rails_helper"
+
+RSpec.describe Guest, type: :model do
+  describe "relationships" do
+    it { should have_many :rooms }
```

Comment:
There should be 2 relationship tests here: `it { should have_many :guest_rooms}` AND `it { should have_many(:guests).through(:guest_rooms) }`